Documentation/ReadStores.md

```diff
@@ -0,0 +1,109 @@
+# Read model stores
+
+In order to create query handlers that perform and enable them search across
+multiple fields, read models or projects are used.
+
+Read models are a flatten views of a subset or all aggregate domain events
+created specifically for efficient queries.
```

Comment:
+ by denormalizing the stored projection to avoid the cost of table and index scans when joining tables in sql.